camelot/admin/entity_admin.py

```diff
@@ -113,7 +113,14 @@ class Admin( EntityAdmin ):
     copy_exclude = []
     validator = EntityValidator
     basic_search = True
-
+    
+    # Temporary hack to allow admins of target entities in one2many/many2many relations to register themselves as editable
+    # with a pending owning instance.
+    # This should be used with extreme care though, as the default list actions only support a persistent owning instance
+    # and thus specialized actions should be used by the target admin to handle the persistence flow correctly.
+    # This is a temporary measure in order to work towards supporting this behaviour in general in the future.
+    allow_relation_with_pending_owner = False
```

Comment:
hoewel ik niks beter kan bedenken, heb ik bedenkingen bij de terminologie 'owner'

het kan ook zijn dat één entity in meerdere one2many relaties voorkomt, en in feite geldt dit dan voor elk van die relaties

Reply:
Ja, het is meer in de zin van eigenaar van de relatie. Dus elke eigenaar entiteit die een relatie naar een entiteit bezit waarbij een admin gebruikt wordt met die vlag geactiveerd laat die relatie toe om editeerbaar te zijn als de eigenaar nog niet persistent is.